programming/sr/motors/index.md

```diff
@@ -57,21 +61,55 @@ R.motor_boards["srXYZ1"].motors[0].power = -1
 
 # motor board srABC1, channel 1 to half power forward
 R.motor_boards["srABC1"].motors[1].power = 0.5
+~~~~~
+
+The motor board will continue to output the requested power until it is told
+otherwise or until power to the board is removed (usually when the robot turns
+off).
 
+Therefore to stop your motors you must explicitly set the power output to zero:
+
+~~~~~ python
 # motor board srXYZ1, channel 0 stopped
 R.motor_boards["srXYZ1"].motors[0].power = 0
 
-# the following will put motor board srABC1, channel 1 at 25% power (forwards) for 2.5 seconds:
+# Put motor board srABC1, channel 1 at 25% power for 2.5 seconds:
 R.motor_boards["srABC1"].motors[1].power = 0.25
 time.sleep(2.5)       # wait for 2.5 seconds
 R.motor_boards["srABC1"].motors[1].power = 0
 ~~~~~
 
+Since each output channel can be controlled separately, you can control several
+motors at once.
+
+~~~~~ python
+# Set one motor to full power in one direction and
+# another to full power in the other:
+R.motor_boards["srABC1"].motors[0].power = 1
+R.motor_boards["srABC1"].motors[1].power = -1
+
+# Wait a while (perhaps for the robot to move)
+time.sleep(3)
+
+# Stop both motors
+R.motor_boards["srABC1"].motors[0].power = 0
+R.motor_boards["srABC1"].motors[1].power = 0
+~~~~~
+
+<div class="warning" markdown="1">
```

Comment:
Hrm, this miiight be better as just an `info`. I'd hoped that `warning` would be amber (it's actually red). I wonder if we should separate "warning" from "danger" (using amber & red)?

Reply:
I think info would work well for this

Reply:
```suggestion
<div class="info" markdown="1">
```